app/models/user_role.rb

```diff
@@ -30,8 +30,16 @@ def self.facility_management_roles
     facility_roles - [FACILITY_STAFF, FACILITY_SENIOR_STAFF]
   end
 
+  def self.global_roles
+    account_manager + administrator + billing_administrator
+  end
+
   def self.valid_roles
-    account_manager + administrator + billing_administrator + facility_roles
+    global_roles + facility_roles
+  end
+
+  def self.global
+    where(facility_id: nil)
```

Comment:
Having said that, we're defining roles by name otherwise so it makes sense to query by name here.